src/lib/validators.ts

```diff
@@ -47,6 +47,21 @@ export async function validateFilePath(
 	}
 }
 
+export async function validatePath(
+	path: string,
+	raise = true,
+): Promise<boolean> {
+	try {
+		await statAsync(path);
```

Comment:
Use async not sync (and actually `await`ing a sync method is pointless :) ).

Reply:
This segment is awaiting for an async function ( **statAsync** ), not a sync one ;)

Reply:
Validators are dropped, as we are now using schema everywhere.